<launch>
  <arg name="delay" default="0.0"/>
  <arg name="record_data" default="true"/>
  <arg name="data_directory" default="/home/joaomendes/real_rosbags/results/knbv"/>
  <arg name="data_frequency" default="2.0" />
  <arg name="time_limit" default="20" />

  <arg name="UAV_NAME" default="$(optenv UAV_NAME uav2)" />
  <arg name="world_frame" default="map" />

  <group ns="$(arg UAV_NAME)">

    <!-- Experiment launcher -->
    <node name="eval_data_node" pkg="motion_planning" type="eval_data_node_real.py" output="screen" required="true">
      <remap from="points" to="/$(arg UAV_NAME)/rgbd_front_pitched/aligned_depth_to_color/points" />
      <remap from="~simulation_ready" to="simulation_ready" />
      <param name="ns_planner" value="planner_node" />
      <param name="ns_voxblox" value="voxblox_node" />
      <param name="delay" value="$(arg delay)" />
      <param name="startup_timeout" value="90.0" />
      <param name="evaluate" value="$(arg record_data)" />
      <param name="eval_directory" value="$(arg data_directory)" />
      <param name="eval_frequency" value="$(arg data_frequency)" />
      <param name="time_limit" value="$(arg time_limit)" />
    </node>
  </group>

</launch>
